tests/fakes/fake_blaise_service.py

```diff
@@ -34,12 +34,14 @@ def add_case_to_questionnaire(
         questionnaire: str,
         case_id: str,
         outcome_code: int = None,
-        wave: str = None,
+        wave: int = None,
```

Comment:
This should break the type checker. If it can be set to `None` then the type should be `Optional[int]`

Reply:
It doesn't but happy to change

Reply:
Either I'm misunderstanding something or mypy is misconfigured 🤣

Reply:
It didn't but changed the way it works in https://github.com/ONSdigital/blaise-totalmobile-services/pull/194